cmd/helmvm/install.go

```diff
@@ -412,50 +443,33 @@ var installCommand = &cli.Command{
 			metrics.ReportApplyFinished(c, err)
```

Comment:
We need to update the comment at line `388` to reflect the new way we install the add-ons.